src/crawlee/_autoscaling/autoscaled_pool.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         is_task_ready_function: Callable[[], Awaitable[bool]],
         is_finished_function: Callable[[], Awaitable[bool]],
     ) -> None:
-        """A default constructor.
+        """Create a new instance.
```

Comment:
`__init__` is not a constructor, it does not create the instance, it initializes it :slightly_smiling_face: Also, I'd prefer to just leave this blank, but I guess it's not possible to just have the `Args` section in a docblock.

Reply:
I also prefer leaving it blank, but as you mentioned, we use it for the args doc blocks.

What options do we have:
- Leave it blank and add a Ruff ignore (if our doc tooling supports it).
- Use a generic placeholder like "Create a new instance.", or maybe better as you pointed out: "Initialize a new instance.".

Reply:
Yeah, b) with the updated wording is OK in my opinion.

Reply:
Resolved